src/integrationTest/java/eu/xenit/gradle/testrunner/ExampleRunner.java

```diff
@@ -30,7 +30,7 @@ public void testExampleDockerPlugin() throws IOException {
 
     @Test
     public void testComposeUp() throws IOException {
```

Comment:
shouldn't we rename the method if we use integrationTest instead of composeUp

Reply:
It tests composeUp, but I just created an other task that is wrapped with `dockerCompose.isRequiredBy`, so it also does a composeDown